batch_processor.py

```
@@ -160,30 +164,59 @@ def _load_prompt(self, filename: str) -> str:
+                    delay = 60 + (attempt * 30)  # 第一次60秒，第二次90秒
+                    print(f"🚫 API限流 (尝试 {attempt + 1}/{max_retries}): {e}")
+                    print(f"⏳ 等待 {delay} 秒后重试...")
+                    time.sleep(delay)
+                else:
+                    # 其他错误使用指数退避
+                    if attempt == max_retries - 1:
+                        print(f"❌ API调用最终失败: {e}")
+                        return None
+                    
+                    delay = min(2 ** attempt, 30)
+                    jitter = random.uniform(0, delay * 0.1)
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)